shared/src/business/useCases/getCaseDeadlinesInteractor.js

```diff
@@ -19,6 +19,9 @@ const { UnauthorizedError } = require('../../errors/errors');
 exports.getCaseDeadlinesInteractor = async ({
   applicationContext,
   endDate,
+  from,
```

Comment:
The params list should be updated in the comments, and `the promise of the getCaseDeadlinesByDateRange call` should say `getCaseDeadlinesInteractor` on line 17.

Reply:
Updated!